README.md

```diff
@@ -1,2 +1,104 @@
-# module-ballerinax-stripe
+# Ballerina Stripe connector
+
+[![Build](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/ci.yml)
+[![Trivy](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/trivy-scan.yml)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-stripe/actions/workflows/build-with-bal-test-native.yml)
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-stripe.svg)](https://github.com/ballerina-platform/module-ballerinax-stripe/commits/master)
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/stripe.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%stripe)
+
 Stripe offers payment processing software and application programming interfaces for e-commerce websites and mobile applications.
+
+The Stripe API is organized around REST. Our API has predictable resource-oriented URLs, accepts form-encoded request bodies, returns JSON-encoded responses, and uses standard HTTP response codes, authentication, and verbs.
```

Comment:
is this correct? shouldn't we mention about the supported stripe API version and information in this section instead?

Reply:
+1